examples/react-17/framer-motion/src/main.tsx

```diff
@@ -1,7 +1,7 @@
 import { OverlayProvider } from 'overlay-kit';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Demo } from './demo';
+import { Demo } from './demo.tsx';
```

Comment:
I have a question.

Since `moduleResolution` is set to `bundler`, it seems like the `.tsx` extension wouldn’t be necessary.
Could you help me understand why it might have been included?

Reply:
Ah! That's not necessary chage right!